src/chrome/content/rules/General-anzeiger-bonn.de.xml

```diff
@@ -1,14 +1,8 @@
-
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://epaper.general-anzeiger-bonn.de/ => https://epaper.general-anzeiger-bonn.de/: (51, "SSL: no alternative certificate subject name matches target host name 'epaper.general-anzeiger-bonn.de'")
-
--->
-<ruleset name="General-anzeiger-bonn.de" default_off='failed ruleset test'>
+<ruleset name="General-anzeiger-bonn.de">
 	<target host="general-anzeiger-bonn.de" />
 	<target host="www.general-anzeiger-bonn.de" />
 	<target host="trauer.general-anzeiger-bonn.de" />
-	<target host="epaper.general-anzeiger-bonn.de" />
+	<target host="epaper.ga-bonn.de" />
```

Comment:
You should add a redirect from `epaper.general-anzeiger-bonn.de` to `epaper.ga-bonn.de` and a comment about the issue.
Please try to add all hosts in `*.ga-bonn.de`.

Reply:
Why do I need to add a redirect? This is already done on the server side. If I open http://epaper.general-anzeiger-bonn.de/ i am redirected to https://epaper.ga-bonn.de/

Reply:
The redirect is made though unsecured http, so you could hijack it. We want HTTPS Everywhere.